delfin/coordination.py

```diff
@@ -350,3 +350,53 @@ def register_watcher_func(self, on_node_join, on_node_leave):
 
     def watch_group_change(self):
         self.coordinator.run_watchers()
+
+
+class GroupMembership(Coordinator):
+
+    def __init__(self, agent_id):
+        super(GroupMembership, self). \
+            __init__(agent_id=agent_id, prefix="")
+
+    def create_group(self, group):
+        try:
+            self.coordinator.create_group(group.encode()).get()
+        except coordination.GroupAlreadyExist:
+            LOG.info("GROUP {0} already exist".format(group))
+
+    def delete_group(self, group):
+        try:
+            self.coordinator.delete_group(group.encode()).get()
+        except coordination.GroupNotCreated:
+            LOG.info("GROUP {0} Group not created".format(group))
```

Comment:
First letter of first word should be in upper case, others should be in lower case.
Same as other log

Reply:
Done